diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
index 31e3da9ff3b4fa1ebb69434b8d42f9548e9ff7a2..17f20df3893784a543fcb36dab1b3fef8fa6a7f4 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -1488,7 +1488,13 @@ void TabDragController::AttachToNewContext(
 
       const bool pinned = detached_tab_collection->get()->pinned_;
 
-      if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
+      if (std::holds_alternative<
+              std::unique_ptr<tabs::TreeTabNodeTabCollection>>(
+              detached_tab_collection->get()->collection_)) {
+        collection_indices =
+            attached_context_->GetTabStripModel()->InsertDetachedTreeTabNodeAt(
+                std::move(*detached_tab_collection), index);
+      } else if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
               detached_tab_collection->get()->collection_)) {
         collection_indices =
             attached_context_->GetTabStripModel()->InsertDetachedTabGroupAt(
