diff --git a/chrome/browser/ui/messages/android/BUILD.gn b/chrome/browser/ui/messages/android/BUILD.gn
index 8076af836e8ec1a9f0064618cd2ec7f85b8ec9a3..bd526f8c73a17e29f165e838226cc0b867cb51c9 100644
--- a/chrome/browser/ui/messages/android/BUILD.gn
+++ b/chrome/browser/ui/messages/android/BUILD.gn
@@ -30,6 +30,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarSwipeHandler.java",
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java",
   ]
+  sources += brave_browser_ui_messages_java_sources
 
   deps = [
     ":java_resources",
