diff --git a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
index d2ce4bf4fb9b3d28afdc33f0c8156c6d6b5b16a1..b8182083c7e4b68c5be60a53362601c9bb534711 100644
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -777,7 +777,7 @@ BrowserViewTabbedLayoutImpl::CalculateProposedLayout(
   }
 
   BrowserLayoutParams params = layout_data_->revised_params;
-  bool needs_exclusion = true;
+  bool needs_exclusion = !delegate().ShouldShowVerticalTabs();
   const HorizontalLayout& horizontal_layout = layout_data_->horizontal_layout;
 
   if (IsParentedTo(views().organizer_tray, views().browser_view)) {
