diff --git a/chrome/browser/ui/views/tabs/groups/tab_group_editor_bubble_view.cc b/chrome/browser/ui/views/tabs/groups/tab_group_editor_bubble_view.cc
index 875bf46862dcb02012bceb937510fb558a15343d..67a8be3cd988fb7d84d5d2edb27eb5186ba51172 100644
--- a/chrome/browser/ui/views/tabs/groups/tab_group_editor_bubble_view.cc
+++ b/chrome/browser/ui/views/tabs/groups/tab_group_editor_bubble_view.cc
@@ -270,6 +270,9 @@ std::unique_ptr<views::Widget> TabGroupEditorBubbleView::Show(
 
   std::unique_ptr<views::Widget> widget =
       BubbleDialogDelegate::CreateBubble(tab_group_editor_bubble_view);
+  MaybeRemoveFooter(tab_group_editor_bubble_view,
+                    tab_group_editor_bubble_view->footer_.ExtractAsDangling());
+  tab_group_editor_bubble_view->footer_ = nullptr;
   tab_group_editor_bubble_view->set_adjust_if_offscreen(true);
   tab_group_editor_bubble_view->GetBubbleFrameView()
       ->SetPreferredArrowAdjustment(
