diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index a21773edf33ab1f016ee67e90d0a3a4ef00569df..8e9747c799fe75a8c03032c3b50416a3a77fbb05 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -22,6 +22,7 @@ source_set("browser_window") {
     "//ui/gfx",
   ]
   deps = [
+    "//brave/browser/ui/browser_window",
     "//components/sessions:session_id",
     "//ui/base/unowned_user_data",
   ]
