diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
index 2a381e14f3403657f3a2dab8edb9ff1ba123946c..19cf3e195210c22d82ff12b64fdff54fdb8c1498 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
@@ -269,6 +269,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             Action.TRANSLATE,
             Action.CREATE_QR_CODE,
             Action.ASK_GEMINI,
+            Action.COPY_CLEAN_LINK,
         })
         @Retention(RetentionPolicy.SOURCE)
         public @interface Action {
@@ -332,7 +333,8 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             int TRANSLATE = 57;
             int CREATE_QR_CODE = 58;
             int ASK_GEMINI = 59;
-            int NUM_ENTRIES = 60;
+            int COPY_CLEAN_LINK = 60;
+            int NUM_ENTRIES = COPY_CLEAN_LINK + 1;
         }
 
         // LINT.ThenChange(/tools/metrics/histograms/enums.xml:ContextMenuOptionAndroid)
@@ -721,6 +723,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             if (!MailTo.isMailTo(mParams.getLinkUrl().getSpec())
                     && !UrlUtilities.isTelScheme(mParams.getLinkUrl())) {
                 linkGroup.add(createListItem(Item.COPY_LINK_ADDRESS));
+                linkGroup.add(createListItem(Item.COPY_CLEAN_LINK));
                 if (!mParams.getLinkText().trim().isEmpty() && !mParams.isImage()) {
                     linkGroup.add(createListItem(Item.COPY_LINK_TEXT));
                 }
