diff --git a/chrome/browser/ui/browser_actions.cc b/chrome/browser/ui/browser_actions.cc
index 3fed7124bddf5077631443d7f81f6b6cafd9e6f1..464b5778c112cb69a8fd8c9bc2a9f7d62179f209 100644
--- a/chrome/browser/ui/browser_actions.cc
+++ b/chrome/browser/ui/browser_actions.cc
@@ -2287,7 +2287,7 @@ void BrowserActions::InitializeToolbarAndMiscActions() {
             .SetTooltipText(l10n_util::GetStringUTF16(IDS_APPMENU_TOOLTIP))
             .SetImage(ui::ImageModel::FromVectorIcon(
                 features::IsRoundedIconsEnabled() ? vector_icons::kMicFilledIcon
-                                                  : vector_icons::kMicOldIcon,
+                                                  : vector_icons::kMicFilledIcon,
                 ui::kColorIcon, ui::SimpleMenuModel::kDefaultIconSize))
             .SetProperty(
                 actions::kActionItemPinnableKey,
@@ -4904,7 +4904,7 @@ void BrowserActions::InitializeToolbarAndMiscActions() {
             vector_icons::kAudioSparkIcon,
 #else
             features::IsRoundedIconsEnabled() ? vector_icons::kMicFilledIcon
-                                              : vector_icons::kMicOldIcon,
+                                              : vector_icons::kMicFilledIcon,
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
             /*is_pinnable=*/true)
             .Build());
