diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
index 36e75f61b7153ec6b01690e6da358fb2fca848b5..571ee78fffff617c597c2ac89ae3d56db4f26545 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
@@ -587,7 +587,7 @@ public class TabSwitcherPaneCoordinator implements BackPressHandler {
                     recyclerViewTimer.getElapsedMillis());
 
             mTabListEditorManager =
-                    new TabListEditorManager(
+                    new BraveTabListEditorManager(
                             activity,
                             mModalDialogManager,
                             coordinatorView,
