diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index d5421fa99f26b4d2d63ffe544a6697c4d6b6318c..5b0894d4d8e4082e3f48d9050342bc700ab8d5bf 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -1994,7 +1994,7 @@ public class ToolbarManager
         }
     }
 
-    private boolean shouldSuppressToolbarLongPress() {
+    protected boolean shouldSuppressToolbarLongPress() {
         return mOmniboxFocusStateSupplier.get()
                 || (mToolbarPositionController != null
                         && mToolbarPositionController.doesPrefMismatchPosition())
