diff --git a/chrome/browser/notifications/BUILD.gn b/chrome/browser/notifications/BUILD.gn
index 0d59dbad53fc3957d2b32d7af4100fb01be4e0e8..8814d055b84ad560e157c9ee69f8989c74df2669 100644
--- a/chrome/browser/notifications/BUILD.gn
+++ b/chrome/browser/notifications/BUILD.gn
@@ -654,6 +654,7 @@ if (is_android) {
     deps = [
       ":java_resources",
       "//base:base_java",
+      "//brave/browser/notifications/android:brave_java",
       "//build/android:build_java",
       "//chrome/android:base_module_java",
       "//chrome/browser/flags:java",
