diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index 46923b3b351741a28f3458d8bd6704f02e71c260..cee2cf3aa69b0a8e10d82bd497ab96746642c7fd 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -651,6 +651,7 @@ bool ConvertedHitTest(views::View* src, views::View* dst, gfx::Point* point) {
   DCHECK(src);
   DCHECK(dst);
   DCHECK(point);
+  BRAVE_BROWSER_VIEW_LAYOUT_CONVERTED_HIT_TEST
   views::View::ConvertPointToTarget(src, dst, point);
   return dst->HitTestPoint(*point);
 }
@@ -1997,6 +1998,7 @@ void BrowserView::OnActiveTabChanged(content::WebContents* old_contents,
       !browser_->GetTabStripModel()->closing_all() && GetWidget()->IsActive() &&
       GetWidget()->IsVisible() && !tab_change_in_split_view;
 #endif
+  BRAVE_BROWSER_VIEW_ON_ACTIVE_TAB_CHANGED
   // Update various elements that are interested in knowing the current
   // WebContents.
 
