diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 62da38252ea57deee09e0a995869b9a4d1b8ffa7..7ac8e71532553f3647a3ceebe721c0d0764c93e3 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5084,6 +5084,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
