diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 1902731a82205cfde77aeb90f667219ca244dab4..8a441049a3b65f8746d58fec23675a066e127c65 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -679,7 +679,7 @@ class BrowserView : public BrowserWindow,
   views::ClientView* CreateClientView(views::Widget* widget) override;
   views::View* CreateOverlayView() override;
 #if BUILDFLAG(IS_MAC)
-  views::View* CreateMacOverlayView();
+  virtual views::View* CreateMacOverlayView();
 #endif
   void OnWindowBeginUserBoundsChange() override;
   void OnWindowEndUserBoundsChange() override;
