diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index 1d0bd3e82b71c8f1205dd7d03ab6cdd6b29cd0d0..e9d59110e55b10440a87769482aca9e43e8825af 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD - style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/ui/views/side_panel/sources.gni")
+
 assert(is_win || is_mac || is_linux || is_chromeos)
 
 import("//chrome/common/features.gni")
@@ -92,6 +94,7 @@ source_set("side_panel") {
     "//ui/webui/resources/cr_components/history_embeddings:mojo_bindings",
     "//url",
   ]
+  public_deps += brave_browser_ui_views_side_panel_public_deps
   deps = [
     "//base:i18n",
     "//chrome/app:generated_resources",
@@ -177,6 +180,7 @@ source_set("side_panel") {
     "//ui/gfx/geometry:geometry_skia",
     "//ui/webui",
   ]
+  deps += brave_browser_ui_views_side_panel_deps
 
   if (enable_extensions_core) {
     public_deps += [
