diff --git a/components/sync/base/features.cc b/components/sync/base/features.cc
index 9c5c02312773221320c84ffa6830b96df3cd5068..7ac0997c71a1d3cfc9f2d0f7c65ec3ea52aca644 100644
--- a/components/sync/base/features.cc
+++ b/components/sync/base/features.cc
@@ -91,6 +91,7 @@ BASE_FEATURE(kReplaceSyncPromosWithSigninPromosNewSignin,
 );
 
 bool IsReplaceSyncPromosWithSignInPromosEnabled() {
+  if ((true)) return false;
   return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
          base::FeatureList::IsEnabled(
              kReplaceSyncPromosWithSigninPromosNewSignin);
@@ -120,7 +121,7 @@ DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAccountSettings);
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
-DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_OVERRIDDEN_FEATURE(kSyncAutofillValuableMetadata, base::FEATURE_DISABLED_BY_DEFAULT);
 
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
 
