diff --git a/components/browser_actuator/internal/BUILD.gn b/components/browser_actuator/internal/BUILD.gn
index 37c713cb2f62e944f1a245f924a7b07f340387d5..a5ca8c6fba27af98648f95c7604197b60892e597 100644
--- a/components/browser_actuator/internal/BUILD.gn
+++ b/components/browser_actuator/internal/BUILD.gn
@@ -34,6 +34,7 @@ source_set("features") {
   ]
 
   visibility = [ "//components/browser_actuator/*" ]
+  visibility += [ "//brave/app:unit_tests" ]
 }
 
 source_set("internal") {
