diff --git a/DEPS b/DEPS
index e3a5d385705194270cbb12f041512144e8fb42dc..4173a0ab10943ff91585653a20168e6e529edea3 100644
--- a/DEPS
+++ b/DEPS
@@ -4533,7 +4533,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
