diff --git a/ios/web/web_state/ui/BUILD.gn b/ios/web/web_state/ui/BUILD.gn
index aa2098b7ec55b4c059a4166a8646302e8e224c00..bd8d2d3bda88e5091d84667cd08270bf606acc7d 100644
--- a/ios/web/web_state/ui/BUILD.gn
+++ b/ios/web/web_state/ui/BUILD.gn
@@ -88,6 +88,7 @@ source_set("ui") {
     "crw_wk_ui_handler.mm",
     "crw_wk_ui_handler_delegate.h",
   ]
+  import("//brave/ios/web/sources.gni") deps += brave_ios_web_web_state_ui_deps
 }
 
 source_set("web_controller_header") {
