diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index 36d5e0df499f264efc890b2bfd537a55e9bf3fc2..52157585cd27afef7970205ab6f7269cd0f55795 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -486,6 +486,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   // Deprecated 08/2026.
   registry->RegisterTimePref(kWaitingForMultiProfileForcedMigrationTimestamp,
                              base::Time());
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -954,6 +955,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // Deprecated 09/2026.
   registry->RegisterDictionaryPref(kSigninHasAcceptedManagementDialog);
+  brave::RegisterBrowserStatePrefs(registry);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -985,6 +987,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
 
   // Added 08/2026.
   prefs->ClearPref(kWaitingForMultiProfileForcedMigrationTimestamp);
+  brave::MigrateObsoleteLocalStatePrefs(prefs);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1052,6 +1055,7 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
 
   // Deprecated 09/2026.
   prefs->ClearPref(kSigninHasAcceptedManagementDialog);
+  brave::MigrateObsoleteProfilePrefs(prefs);
 }
 
 void MigrateObsoleteUserDefault() {
