diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index b51c5aa12fa19dacaa555024fd761af8802405e3..7ae9f15a7806adf58d8322835a49995a271e3cce 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -130,6 +130,7 @@ source_set("flags") {
     "//ios/web/common:web_view_creation_util",
     "//ios/web/public",
   ]
+  import("//brave/ios/browser/flags/sources.gni") deps += brave_flags_deps
 }
 
 source_set("system_flags") {
