diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index b8bfc8d482c40da7ffc4ce6904782df6cb7445b4..2cf223e2c4997b1727c3fe77f29f66c7e42a4cfa 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/ai_chat/sources.gni")
 import("//third_party/blink/renderer/modules/modules.gni")
 
 blink_modules_sources("peerconnection") {
@@ -194,6 +195,7 @@ blink_modules_sources("peerconnection") {
     "//third_party/blink/renderer/modules/permissions",
     "//third_party/blink/renderer/modules/webrtc",
   ]
+  deps += brave_blink_renderer_modules_peerconnection_ai_chat_deps  # Brave
 
   allow_circular_includes_from =
       [ "//third_party/blink/renderer/modules/mediastream" ]
