diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index da8208186bbf5af57865ac69c6dd535e4646d6d4..3116bc0437b9a1c75ce02f88eeca7c69aa800cfe 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2841,6 +2841,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
