diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index 39522f2982953879e8115662831f6238c2074b6e..ba21d2ce0648e05e4a6a8682fe439d3bbc65a81f 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1811,6 +1811,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return `${host}:${port}`;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 export type RequestConditionsSetting = {
   url: string,
